apps/web/pages/codercamp.tsx

```diff
@@ -0,0 +1,117 @@
+import { Footer, Header } from "@coderdojobraga/ui";
+import { MessageFilled, ScheduleOutlined } from "@ant-design/icons";
+import Image from "next/image";
+
+const CoderCamp = () => (
+  <>
+    <Header landing={true} />
+
+    <div className="container mx-auto mt-12">
+      <h1
+        className="m-4 text-center text-4xl font-bold sm:text-5xl"
+        style={{ fontFamily: "Bankai", fontSize: "5rem", lineHeight: "5rem" }}
+      >
+        Coder Camp
+      </h1>
+      <div className="bg-primary h-8/12 mx-auto w-6/12 py-0.5 text-2xl lg:w-5/12" />
+      <p className="m-4 pt-6 text-center text-3xl font-normal sm:text-3xl">
+        03 julho - 07 julho
+      </p>
+    </div>
+    <section className="overflow-hidden bg-white">
+      <div className="container mx-auto">
+        <div className="flex flex-wrap items-center justify-center">
+          <div className="ml-auto mr-auto mt-20 w-full px-12 md:w-5/12 md:px-4">
+            <div className="bg-primary mb-6 inline-flex h-16 w-16 items-center justify-center rounded-full p-3 text-center text-white shadow-lg">
+              <ScheduleOutlined style={{ fontSize: "150%" }} />
+            </div>
+            <h3 className="mb-2 text-3xl font-semibold leading-normal text-black">
+              O que é o Coder Camp?
+            </h3>
+            <p className="mt-4 mb-4 text-lg font-light leading-relaxed text-black">
+              O Coder Camp, organizado pelo CoderDojo Braga, é uma semana
+              dedicada a atividades lúdicas no âmbito da programação, realizada
+              entre os dias 3 a 7 de julho. Esta iniciativa promove a
+              aprendizagem da programação num contexto informal e de convívio,
+              na companhia de outros Ninjas e dos nossos Mentores.
+            </p>
+            <a
+              href=""
+              className="bg-primary mt-4 mr-1
+                                       mb-1 inline-block rounded-3xl px-6 py-4
+                                       text-center text-sm font-bold uppercase text-white shadow
+                                       outline-none hover:bg-purple-600 hover:text-white"
+            >
+              Inscrições abrem em breve
+            </a>
```

Comment:
Maybe now I would remove this part once that now there's no need to ave it and I can't really think of another usage for it.
```suggestion
```